src/Hal/Metric/Consolidated.php

```diff
@@ -119,22 +119,24 @@ public function __construct(Metrics $metrics)
         $avg->outgoingCDep = 0;
         $avg->outgoingPDep = 0;
         $avg->classesPerPackage = 0;
-        foreach (array_keys($packages) as $eachName) {
-            /* @var $eachPackage PackageMetric */
-            $eachPackage = $metrics->get($eachName);
-            $avg->distance += $eachPackage->getDistance();
-            $avg->incomingCDep += count($eachPackage->getIncomingClassDependencies());
-            $avg->incomingPDep += count($eachPackage->getIncomingPackageDependencies());
-            $avg->outgoingCDep += count($eachPackage->getOutgoingClassDependencies());
-            $avg->outgoingPDep += count($eachPackage->getOutgoingPackageDependencies());
-            $avg->classesPerPackage += count($eachPackage->getClasses());
+        if ([] !== $packages) {
```

Comment:
would advice to use `count($packages) !== 0`

Reason is that this ensures type safety.. meaning that it ensures that `$packages` is an array..

Reply:
In that case, I'll use `$sum->nbPackages !== 0` as this was already the result of the count.
Actually, because there's already `$sum->nbPackages = count($packages)` earlier in the same snippet, it is already sure that `$package` is an array. So can't we keep this condition?

If you prefer `$sum->nbPackages !== 0`, I'm ok too.